control-plane/connect-inject/container_init.go

```diff
@@ -117,12 +117,16 @@ func (h *Handler) containerInit(namespace corev1.Namespace, pod corev1.Pod) (cor
 		return corev1.Container{}, err
 	}
 
-	var consulDNSIP string
+	var consulDNSClusterIP string
 	if dnsEnabled {
+		// If Consul DNS is enabled, we loop through the environment variables
+		// to identify the one that has the value of the ClusterIP onf the Consul
+		// DNS Service. constructDNSServiceHostName returns the name of the env
+		// variable whose value is the ClusterIP of the Consul DNS Service.
 		for _, e := range os.Environ() {
```

Comment:
Is this looping still necessary now that the env var is fully specified?

Reply:
good catch!!